migrations/versions/0500_update_intl_limit_values.py

```diff
@@ -0,0 +1,84 @@
+"""
+Create Date: 2025-04-29 12:36:49.360512
+"""
+
+import textwrap
+
+from alembic import op
+from flask import current_app
+from sqlalchemy import text
+
+revision = '0500_update_intl_limit_values'
+down_revision = '0499_merge_callback_tables'
+
+
+def upgrade():
+    conn = op.get_bind()
+
+    # First get services that sent international SMS in the last 12 months, and how many they sent on their busiest day.
+    # Then for each of those services, if they sent more than 50 on their busiest day,
+    #  set their new international_sms_message_limit to busiest day * 2.
+
+    # Then get all remaining services that have their international_sms_message_limit set to default value (250,000)
+    # and set their limit to 100 per day.
+
+    services_with_intl_sms = conn.execute(
+        text(
+            textwrap.dedent(
+                """
+                SELECT intl_billing.service_id, MAX(intl_billing.intl_sent) AS max_daily_intl
+                FROM (SELECT
+                    service_id, bst_date, SUM(notifications_sent) as intl_sent FROM ft_billing WHERE bst_date > '2024-04-21' AND notification_type = 'sms' AND international = true GROUP BY service_id, bst_date
```

Comment:
Should this date be hardcoded?

Do we want the upgrade to target services in the last 12 months from whenever the upgrade is run or do we want it to target services for a fixed date range `2024-04-21` to `2025-04-20`?

Reply:
I think the start date doesn't matter too much, and the query should look until the present moment at the time of running.

My reasoning: we want to get services busiest day in recent time - it doesn't really matter if it's in last 12 or 13 months, but it does matter that we don't skip ant recent days.

So this query meets both these conditions. I don't feel very strongly about it though. I just like the simplicity of it.

Reply:
Tbh then that's fine with me. It's just about catching _enough_ people who are using it somewhat recently.